### PR TITLE
Fix(refs:35060): remove required symbol for emailAddress

### PR DESCRIPTION
**Pull request by elmasdemos**
**Ticket:** https://yaits.demos-deutschland.de/T35060

### Description: 

- the email field should not have the mandatory symbol, because its not mandatory here
- the phoneNumber field shall be reqquired

### How to review/test
- login as mandant admin and look how the label for email field looks like when you wanna create a new support contact
- try to save a contact without phoneNumber. you should get an error and it should fail

### PR Checklist

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: translations/messages+intl-icu.de.yml [hunk 1]
```diff
@@ -1029,6 +1029,7 @@ error.copy.mapfile: "Die Einzeichnung der Stellungnahme {externId} konnte nicht
 error.changes.not.saved: "Beim Speichern der Änderungen ist ein Fehler aufgetreten."
 error.date.authored: "Bitte geben Sie ein Verfassungsdatum ein."
 error.in.fields: "Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: "
+error.phone.required: "Bitte geben sie eine gültige Telefonnummer ein."
```
Comment (muellerdemos) [inline translations/messages+intl-icu.de.yml:1032]:
```suggestion
error.phone.invalid: "Bitte geben sie eine gültige Telefonnummer ein."
```
I think the key should transport the info that the phone number is invalid

File: client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue [hunk 2]
```diff
@@ -41,14 +41,14 @@
             autocomplete="tel"
             class="u-mb-0_75"
             data-cy="phoneNumber"
-            :data-dp-validate-error="customerContact.eMailAddress === ''
-              ? 'error.phone_or_email.required'
+            :data-dp-validate-error="customerContact.phoneNumber === ''
+              ? 'error.phone.required'
               : 'error.phone.pattern'"
```
Comment (muellerdemos) [inline client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue:46]:
```suggestion
           :data-dp-validate-error="!customerContact.phoneNumber ? 'error.phone.required' : 'error.phone.pattern'"
```
If you write it like that it is readable enough to put it in one line which would improve the overall readability imho.

**Review by muellerdemos — CHANGES_REQUESTED** [review]
I see minor adjustments possible to improve code readability. Also this PR has `main` as its target branch, is this correct? I think the target should be `release`

File: translations/messages+intl-icu.de.yml (hunk 1 above)
Comment (elmasdemos) [reply translations/messages+intl-icu.de.yml:1032]: here it has to be required name cause the other errors have the same name and the focus is that its required. the focus is not on invalid

File: client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue (hunk 2 above)
Comment (elmasdemos) [reply client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue]: done https://github.com/demos-europe/demosplan-core/pull/2156/commits/8a2b9fbbb0a9234ca82acf496b764afa254f9c69

File: translations/messages+intl-icu.de.yml (hunk 1 above)
Comment (muellerdemos) [reply translations/messages+intl-icu.de.yml:1032]: But we want to have generic translations hence the naming of the translation key should not be dependent on a specific area in the code. Or the wording of the key must be adjusted imho

File: translations/messages+intl-icu.de.yml [hunk 3]
```diff
@@ -1029,6 +1029,7 @@ error.copy.mapfile: "Die Einzeichnung der Stellungnahme {externId} konnte nicht
 error.changes.not.saved: "Beim Speichern der Änderungen ist ein Fehler aufgetreten."
 error.date.authored: "Bitte geben Sie ein Verfassungsdatum ein."
 error.in.fields: "Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: "
+error.phone.required: "Bitte geben Sie eine Telefonnummer ein."
 error.phone_or_email.required: "Bitte geben Sie entweder eine gültige E-Mail-Adresse oder Telefonnummer ein."
 error.phone.pattern: "Bitte geben Sie gültige Telefonnummer ein."
```
Comment (muellerdemos) [inline translations/messages+intl-icu.de.yml:1034]: nitpick: not in alphabetic sort order

Comment (elmasdemos) [reply translations/messages+intl-icu.de.yml]: done https://github.com/demos-europe/demosplan-core/pull/2156/commits/a608bf8147a90024fda1261b1937f04e41da45eb

File: client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue (hunk 2 above)
Comment (elmasdemos) [reply client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue]: ah, I seedone https://github.com/demos-europe/demosplan-core/pull/2156/commits/4e5a64bd4000ddeb2b234fbe8925ab94b191600a

File: translations/messages+intl-icu.de.yml (hunk 1 above)
Comment (elmasdemos) [reply translations/messages+intl-icu.de.yml]: done https://github.com/demos-europe/demosplan-core/pull/2156/commits/4e5a64bd4000ddeb2b234fbe8925ab94b191600a

**Review by muellerdemos — APPROVED** [review]
thanks code lgtm

**Pull request merged**